docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG NODE_VERSION=8.12.0-slim
+ARG NODE_VERSION=8.15.0-slim
```

Comment:
Please tell about the change in  CHANGES_NEXT_RELEASE file. Something like this:

```
- Upgrade NodeJS version from 8.12.0 to 8.15.0 in Dockerfile due to security issues
```

Reply:
Fixed #13cddf9